lab_1/main.py

```diff
@@ -69,7 +108,41 @@ def get_concordance(tokens: list, word: str, left_context_size: int, right_conte
     right_context_size = 3
     --> [['man', 'is', 'happy', 'the', 'dog', 'is'], ['dog', 'is', 'happy', 'but', 'the', 'cat']]
     """
-    pass
+    conco_all = []
+    index_word = 0
+
+    if checker(tokens, list) and checker(word, str) \
+            and checker(left_context_size, int) and checker(right_context_size, int):
+        normal_concordance = left_context_size >= 0 and right_context_size >= 0
+
+        if "".join(tokens) != word and word in tokens and normal_concordance:
+            index_word = tokens.index(word, index_word, len(tokens))
+
+            for i in range(tokens.count(word)):
+                try:
+                    concordance = tokens[index_word - left_context_size: index_word] \
+                                  + tokens[index_word: index_word + right_context_size + 1]
+                except IndexError:
```

Comment:
May be it is better to use some condition here?

Reply:
Why is it worse to use try except?